cosmo_tester/test_suites/cluster/config/haproxy.cfg

```diff
@@ -0,0 +1,24 @@
+global
+    maxconn 100
+    tune.ssl.default-dh-param 2048
+defaults
+    log global
+    retries 2
+    timeout client 30m
+    timeout connect 4s
+    timeout server 30m
+    timeout check 5s
+listen manager
+    bind *:80
+    bind *:443 ssl crt /etc/haproxy/cert.pem
+    redirect scheme https if !{ ssl_fc }
+    mode http
+    option forwardfor
+    stick-table type ip size 1m expire 1h
+    stick on src
+    option httpchk GET /api/v3.1/status HTTP/1.0\r\nAuthorization:\ Basic\ YWRtaW46YWRtaW4=\r\nTenant:\ default_tenant
```

Comment:
Using admin:admin as the basic auth header will cause problems if we change the admin password in any cluster tests. We don't at the moment, but it's conceivably something that could happen. I'd suggest using the config from https://docs.cloudify.co/5.0.5/install_maintain/installation/installing-cluster/#management-service-load-balancer instead, which checks for a 401.